### PR TITLE
boards/cc2538dk: adding PORT_DARWIN for mac support

### PR DESCRIPTION
**Pull request by kYc0o**

### CODE REVIEW

File: boards/cc2538dk/Makefile.include [hunk 1]
```diff
@@ -4,6 +4,7 @@ export CPU_MODEL ?= cc2538nf53
 
 # setup serial terminal
 PORT_LINUX ?= $(firstword $(wildcard /dev/serial/by-id/*XDS100v3*if01* /dev/ttyUSB1))
+PORT_DARWIN ?= $(shell ls -1 /dev/tty.usbserial-* | head -n 1)
```
Comment (Kijewski) [inline boards/cc2538dk/Makefile.include]: Can you use the `$(firstword $(wildcard …))` pattern, so #4913 does not need to change the file, please?

Comment (gebart) [inline boards/cc2538dk/Makefile.include]: Thanks @Kijewski

Comment (kYc0o) [inline boards/cc2538dk/Makefile.include]: Ok I can do that, I wasn't aware of that PR. Thanks!

**Comment by kYc0o** [comment]
Oups, maybe I committed in a wrong way... can you check if everything is ok?

**Comment by LudwigKnuepfer** [comment]
`git rebase -i master`

**Comment by kYc0o** [comment]
I'll discard this and make a new PR

**Comment by LudwigKnuepfer** [comment]
No, not necessary

**Comment by kYc0o** [comment]
Any hint to revert all this mess :/

**Comment by LudwigKnuepfer** [comment]
yes

**Comment by kYc0o** [comment]
now it should be ok, someone can ACK?

**Comment by emmanuelsearch** [comment]
It compiles but does not flash and I get the warning below. Any clues?

```
Warning: no PORT set!
```

**Comment by emmanuelsearch** [comment]
Worked after disabling the signature checks and loading an older FTDI driver.

**Comment by emmanuelsearch** [comment]
ACK

**Comment by haukepetersen** [comment]
and go.

**Pull request merged**